Framework/Intersect.Framework.Core/Color.cs

```diff
@@ -306,7 +317,29 @@ public static Color FromRgba(int rgba)
         return new Color(parts[0], parts[1], parts[2], parts[3]);
     }
 
-    public static implicit operator Color(string colorString) => FromString(colorString);
+    public static Color? FromString(string val, Color? defaultColor = default)
+    {
+        if (string.IsNullOrWhiteSpace(val))
+        {
+            return defaultColor;
+        }
+
+        val = val.Trim().ToLowerInvariant();
+
+        if (KnownColors.TryGetValue(val, out Func<Color>? colorFunc))
+        {
+            return colorFunc();
+        }
+
+        Color? parsedColor = Color.FromHex(val, defaultColor);
+
+        parsedColor ??= Color.FromCsv(val, defaultColor);
```

Comment:
Don't pass `defaultColor` here -- if it's non-null and the string is CSV instead of Hex or Known it would use the `defaultColor` instead of the would-be-valid CSV.

And while there's (currently) no difference between it being or not being passed to `FromCsv()` on L336, don't include it since we're addressing it on L338